src/sentry/web/frontend/debug/debug_chart_renderer.py

```diff
@@ -524,35 +704,133 @@ def get(self, request: HttpRequest) -> HttpResponse:
         ret = []
 
         ret.append(
-            charts.generate_chart(ChartType.SLACK_DISCOVER_TOTAL_PERIOD, discover_total_period)
+            {
```

Comment:
This refactor is mainly to make it easier to see what chart is what in the debug view

Reply:
nittiest nit but can you make Discover capitalized? it makes it clearer it's the product Discover and not just ... Slack discovering something